storage/store.go

```diff
@@ -1007,8 +1045,19 @@ func (s *Store) GroupStorage(groupID uint64) multiraft.WriteableGroupStorage {
 	defer s.mu.RUnlock()
 	r, ok := s.ranges[int64(groupID)]
 	if !ok {
-		log.Warningf("%p requested nonexistent range with raft ID %d", s, groupID)
-		return nil
+		var err error
+		r, err = NewRange(&proto.RangeDescriptor{
+			RaftID: int64(groupID),
```

Comment:
Any reason we're using `int64` while etcd/raft is using `uint64` for IDs? Sort of awkward to have to cast back and forth.